<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteHiddenHelperClass</class>
 <widget class="QMainWindow" name="RemoteHiddenHelperClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemoteHiddenHelper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Integration">
         <attribute name="title">
          <string>Integration Portal</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QPushButton" name="btnFlash">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>GroupBox</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_14">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_13">
                   <item>
                    <widget class="QCheckBox" name="cbxBootloader">
                     <property name="text">
                      <string>BootLoader</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="cbxAC">
                     <property name="text">
                      <string>AC</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="cbxGC">
                     <property name="text">
                      <string>GC</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="lProject">
                <property name="text">
                 <string>Project</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbProject"/>
              </item>
              <item>
               <widget class="QLabel" name="lSamplePhase">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="text">
                 <string>SamplePhase</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbSamplePhase"/>
              </item>
              <item>
               <widget class="QLabel" name="lRelease">
                <property name="text">
                 <string>Release</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbRelease"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QListView" name="listView"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabDragAndDrop">
         <attribute name="title">
          <string>Drag and Drop</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="hbDragLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>File</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="remotehiddenhelper.qrc"/>
 </resources>
 <connections/>
</ui>
